ooniprobe/View/SettingsTableViewController.mm

```diff
@@ -224,8 +230,8 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 {
     if (indexPath.section == 2 && indexPath.row == [advancedItems count] -1){
         NSString *current = [advancedItems objectAtIndex:indexPath.row];
-        UIAlertView *alert = [[UIAlertView alloc] initWithTitle:NSLocalizedString(current, @"") message:nil delegate:self cancelButtonTitle:NSLocalizedString(@"cancel", @"") otherButtonTitles:NSLocalizedString(@"ok", nil), nil];
-        alert.tag = indexPath.row;
+        UIAlertView *alert = [[UIAlertView alloc] initWithTitle:NSLocalizedString(current, @"") message:nil delegate:self cancelButtonTitle:NSLocalizedString(@"cancel", @"") otherButtonTitles:NSLocalizedString(@"ok", nil), NSLocalizedString(@"set_default", nil), nil];
+        alert.tag = 1;
```

Comment:
- [ ] @bassosimone needs to understand this change

Reply:
So, it seems `indexPath.row` was unused and now you're using a stable tag such that you know that when the tag is equal to one you have clicked to modify the collector?

Reply:
That was a mistake, there was no delagate handling. Now there is only one delegate, to set the default collector.